src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -822,6 +823,8 @@ internal static void ValidateSpecifiedAuthentication(PSCredential credential, st
         private const string KeyFilePathParameter = "KeyFilePath";
         private const string IdentityFilePathAlias = "IdentityFilePath";
         private const string PortParameter = "Port";
+        private const string ConfigurationNameParameter = "ConfigurationName";
```

Comment:
We should remove this parameter since we don't yet support it.

Reply:
Yes, accidentally left behind